test-projects/basic/cypress/integration/relationship-field-re-hydration_spec.js

```diff
@@ -5,7 +5,9 @@ describe('Testing re-hydration', () => {
     cy.visit('/admin/posts');
     cy.get('#list-page-create-button').click();
     cy.wait(150);
-    cy.get('#react-select-ks-input-categories div[aria-hidden="true"]').click();
+    cy.get('#react-select-ks-input-categories div[aria-hidden="true"]')
+      .first()
```

Comment:
Fixes the test failures on CI. I can't replicate locally and it only appears to happen _sometimes_ on CI.

It's particularly annoying because the failure indicates that selector matches 2 elements, which when run locally I can only see a single element 😠

Reply:
It's [still failing](https://circleci.com/gh/keystonejs/keystone-5/17394#artifacts/containers/0) :sob:

Trying a different method which will hopefully work.

Reply:
Errmmmaagghheerrrd, it works 🎉